### PR TITLE
arduino-builder: refresh page

### PR DESCRIPTION
**Pull request by EmilySeville7cfg**
Experimental syntax for ranges was used: #9564.

- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Comment by blueskyson** [comment]
I prefer the original example description.

File: pages/common/arduino-builder.md [hunk 1]
```diff
@@ -8,9 +8,9 @@
 
 `arduino-builder -compile {{path/to/sketch.ino}}`
 
-- Specify the debug level (1 to 10, defaults to 5):
+- Specify the debug level (default: 5):
 
-`arduino-builder -debug-level {{level}}`
+`arduino-builder -debug-level {{1,...,10}}`
```
Comment (kbdharun) [inline pages/common/arduino-builder.md:13]:
```suggestion
`arduino-builder -debug-level {{1,2,3,4,5,6,7,8,9,10}}`
```
How about this? 
Also, I agree with @blueskyson about the command description change.

Comment (EmilySeville7cfg) [reply pages/common/arduino-builder.md:13]: I think we should not present more than `n` numbers in placeholders literally. If there are more than `n` numbers then we can use `from..to` syntax or `from,...,to`. But the point is it should be used everywhere. And rules have to be written to fix this fact.

**Review by blueskyson — APPROVED** [review]
This is clearer than the previous commit.

**Review by kbdharun — APPROVED** [review]

**Pull request merged**